app/frontend/src/signOutAfterInactivity.js

```diff
@@ -0,0 +1,28 @@
+var timeoutInMinutes = 30
+
+var timeoutId; 
+
+function resetTimer() { 
```

Comment:
can we use arrow functions? e.g function blah() {} becomes const blah = () => {}

Reply:
just discussed with @tatyree - dont worry about this newer JS syntax for now

Reply:
@david-mears-dfe, just talked it over with @alexbowen and we're not 100% sure we're transpiling arrow functions (they don't work in IE and need to be changed by Babel), so just leave it as is for now.  Once we're sure, we'll have a clear up of them.